.ci/check

```diff
@@ -48,10 +48,22 @@ fi
 # Install Golint (linting tool).
 go get -u golang.org/x/lint/golint
 
+HELM_URL="https://get.helm.sh"
+HELM_VERSION="v2.14.1"
+HELM_SYSARCH="linux-amd64"
+HELM_BINARY="helm-${HELM_VERSION}-${HELM_SYSARCH}.tar.gz"
+
+# Install Helm from binary.
+curl -O ${HELM_URL}/${HELM_BINARY}
+tar -zxvf ${HELM_BINARY}
+mv ${HELM_SYSARCH}/helm /usr/local/bin/helm
+rm -rf ${HELM_BINARY} ${HELM_SYSARCH}
```

Comment:
Use helm install script, otherwise you will need check for OS and arch. Since we want script to be run locally as well.
```bash
curl -LO https://git.io/get_helm.sh
chmod 700 get_helm.sh
./get_helm.sh --no-sudo
rm get_helm.sh
```

Reply:
Agreed. Done.